APIScrape/scrape.php

```diff
@@ -8,7 +8,8 @@
 $dbConnection = $db->dbConnectToHostOnly();
 
 /**
- * Creates a new database into which to put the scraped data.
+ * @param PDO $db Takes DB containing a connection to the database.
+ *  * Creates a new database into which to put the scraped data.
```

Comment:
Don't bother changing this, it doesn't block merge, but param should go after description:
```
* Creates a new database into which to put the scraped data.
*  
* @param PDO $db Takes DB containing a connection to the database.
```